docker/Dockerfile.hardware

```
@@ -2,21 +2,20 @@ ARG ROS_DISTRO=humble
 
 FROM husarnet/ros:${ROS_DISTRO}-ros-core
 
-ARG BRANCH_NAME=ros2-devel
 ARG BUILD_TEST=OFF
 
 ENV HUSARION_ROS_BUILD_TYPE=hardware
 
 STOPSIGNAL SIGINT
 
 WORKDIR /ros2_ws
+COPY .. /ros2_ws/src/husarion_ugv
```

```suggestion
COPY . src/husarion_ugv
```
I believe this will be run from panther_ros directory so context will be set there. Also no point using full path when we specify WORKDIR